Changelog.md

```diff
@@ -1,3 +1,7 @@
+## FF7
+
+- Core: added option to filter what the original game filtered when still using internal textures.
+
```

Comment:
Please put this section in the right order after the Next. See other PR examples.

Reply:
Sorry for taking time getting back to this. I got sick. still am but now i've actually spotted this.

The original 98 release applied filtering to all the 3d textures.  FFNX, well, doesn't. a blanket fix appears to have been put in to stop it just to remove all the graphical glitches when upscaling.

The steam release does too, but lets you disable it. That's why a new flag is needed, and the ability to let users choose it.